### ISSUE TITLE
Only first value being returned

### ISSUE THREAD

**Issue by dimitarnestorov** [joshstevens19/ethereum-multicall#13]
The function `globalPositionData` of [kovan contract 0xD21d3A321eDc8ca5FEA387A4D082a349c86CCfE5](https://kovan.etherscan.io/address/0xD21d3A321eDc8ca5FEA387A4D082a349c86CCfE5#readContract) returns two values but only one of them is being returned when using ethereum-multicall with web3.js.

After a little bit of debugging the problem seems to be the [`formatReturnValues`](https://github.com/joshstevens19/ethereum-multicall/blob/559657d18198f1808971af4cd4b961cf6b46d3cb/src/multicall.ts#L256) function.

![Screenshot from 2021-08-06 15-03-53](https://user-images.githubusercontent.com/8790386/128507917-cac56d61-3d7f-44c1-801f-abfdb6c22b35.png)
![Screenshot from 2021-08-06 15-04-12](https://user-images.githubusercontent.com/8790386/128507914-cad85e30-1397-445b-9f04-00a5d66749fe.png)

I commented out these lines to fix the issue locally:
```
        if (Array.isArray(decodedReturnResults)) {
            return decodedReturnResults;
        }
```

**Comment by joshstevens19** [comment]
Hey thanks for this issue! hmm, that is odd! I will have a look at this will need to work out if its web3 or ethers as I am sure this works on ethers.

**Comment by joshstevens19** [comment]
Hey this is fixed in 2.7.1 please download the new one and let me know yours works now. Thanks for raising this issue.
